azkaban-exec-server/src/main/java/azkaban/execapp/AzkabanExecutorServer.java

```diff
@@ -314,11 +309,6 @@ private void loadCustomJMXAttributeProcessor(Props props) {
     }
   }
 
-
-  public ProjectLoader getProjectLoader() {
-    return projectLoader;
-  }
-
   public ExecutorLoader getExecutorLoader() {
```

Comment:
I'm seeing that ExecutorServlet calls this method. Should we inject executorLoader there in near future?